service/cloudFormation/sam-template.yml

```diff
@@ -44,7 +44,7 @@ Resources:
       Description: Lambda Function to run playwright test
       FunctionName: !Ref LambdaTestName
       Role: !GetAtt syntheticQueryS3Bucket.Arn
-      MemorySize: 512
+      MemorySize: 1024
       Timeout: 60
```

Comment:
In the template, the timeout is 60, and in the code for creating the lambda function, the timeout is 300. Is this intentional?